src/Http/Session.php

```diff
@@ -155,7 +155,7 @@ protected static function _defaultConfig(string $name): array
                     'session.use_trans_sid' => 0,
                     'session.serialize_handler' => 'php',
                     'session.use_cookies' => 1,
-                    'session.save_path' => defined('TMP') ? TMP : sys_get_temp_dir()
+                    'session.save_path' => (defined('TMP') ? TMP : sys_get_temp_dir())
                         . DIRECTORY_SEPARATOR . 'sessions',
```

Comment:
```suggestion
                    'session.save_path' => (defined('TMP') ? TMP : sys_get_temp_dir() . DIRECTORY_SEPARATOR)
                        . 'sessions',
```

`DIRECTORY_SEPARATOR` needs to be appended only to `sys_get_temp_dir()`; `TMP` already has a trailing slash.